README.md

```diff
@@ -1,2 +1,198 @@
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Single Page Application (SPA)]**
+- **[Nodejs for development]**
+- **[Webpack for static bundler]**
+- **[React: a frontend library]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Check Git is already installed and working:
+
+```sh
+ git --version
+ node --version
+```
+
+It should not show you any error. And should return the version info.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Execute this:
+
+```sh
+  git clone git@github.com:mahabubx7/math-magicians.git
+```
+
+### Install
+
+Install this project with:
+
+Install Dependencies:
+
+```sh
+  cd your-project-folder
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Execute this:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Testing with Lint-checks:
+
+```sh
+  npm test
+```
+
+### Live Preview
+Not available yet.
+
+### Deployment
+
+Deployed in `Github Pages`. [Live Preview](https://mahabubx7.github.io/math-magicians/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mahabub**
+
+- GitHub: [@mahabubx7](https://github.com/mahabubx7)
+- Twitter: [@mahabub\_\_7](https://twitter.com/mahabub__7)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mahabubx7)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Components & Props]**
+- [ ] **[React Router]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project... please give it a start.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank... themealdb.com
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Why Webpack?]**
+
+  - [It is a static bundler. It helps us to bundle a big and massy web-pages project into a bundled and gives us optimized performance. ]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [CC License](./LICENSE) licensed.
```

Comment:
- [ OPTIONAL ] Since the license you used was an `MIT LICENSE`, I will suggest that you change the name of the License from `CC  License` to `MIT License`. This is essential to help provide users who visit your page with clarity and avoids confusion :wink: